frontend/src/components/GameList.vue

```diff
@@ -13,15 +18,22 @@
         title: String,
         author: String,
         description: String,
-        image: String,
+
     },
 
     data: () => ({
       show: false,
+      gameInfo: [1,2,3,4,5,6,7,8]
     }),
 
     components: {
       GameCard,
     },
+
+    methods: {
+      getData() {
+        let res =  this.$axios.get('/game')
```

Comment:
add the line 
`this.gameInfo = res.data[0];`
Also you should probably add async and await